toolkit/tools/pkg/imagecustomizerlib/artifactsinputoutput.go

```diff
@@ -24,47 +26,76 @@ const (
 	SystemdBootDir = "EFI/systemd"
 )
 
+type OutputArtifactMetadata struct {
+	Partition   InjectFilePartition `yaml:"partition"`
```

Comment:
The names of the types seem to use a mix between `Output` and `Inject`. We should use only one. I recommend using `Inject`.